### DISCUSSION TITLE
Regarding converting long texts to Vector Embeddings with Towhee

### DISCUSSION THREAD

**Discussion by ErkMagnifico** [towhee-io/towhee#2592] [Q&A] [answered]
Hello everyone,

When trying to convert long texts into vector embeddings using models like data2vec or dpr, 
![image](https://github.com/towhee-io/towhee/assets/101665766/3bc0414f-9da9-44b5-bbe1-69e461c053b0)

The following error occurs:
![image](https://github.com/towhee-io/towhee/assets/101665766/778822f1-19ca-4874-baeb-0656d712f0be)

In the case of dpr:
![image](https://github.com/towhee-io/towhee/assets/101665766/8836be6c-270a-4d19-87d1-9f6cccd98007)

![image](https://github.com/towhee-io/towhee/assets/101665766/a181954c-9086-40cf-86a3-7694eaea52f0)

For reference, the size of the text that I am passing in is 3382 characters long, with 551 words.

Are there any workarounds or solutions for the same? 
Any help would be appreciated.

**Comment by junjiejiangjjj** [comment, marked as answer]
https://towhee.io/tasks/detail/pipeline/sentence-similarity   You can try out the models here, which can support longer text.

**Comment by ErkMagnifico** [comment]
The model present by default in the AutoPipe worked, thank you!
